src/codehelp/formatter.vala

```diff
@@ -32,26 +32,34 @@ namespace Vls.Formatter {
         var launcher = new SubprocessLauncher (SubprocessFlags.STDERR_PIPE | SubprocessFlags.STDOUT_PIPE | SubprocessFlags.STDIN_PIPE);
         launcher.set_environ (Environ.get ());
         string? stdout_buf, stderr_buf;
-        Subprocess subprocess = launcher.spawnv (get_uncrustify_args (options, analyzed_style));
+        Subprocess subprocess;
+        var argv = get_uncrustify_args (options, analyzed_style);
+        try {
+            subprocess = launcher.spawnv (argv);
+        } catch (Error e) {
+            // FIXME!!! This is a workaround until the flag "SEARCH_PATH_FROM_ENVP" is widely available
+            argv[0] = "/usr/local/bin/uncrustify";
```

Comment:
Don't use this. Instead of hard-coding the location, use `SubprocessLauncher`.

Reply:
You can use `SubprocessLauncher.set_environ()` instead of `SEARCH_PATH_FROM_ENVP` right?
https://valadoc.org/gio-2.0/GLib.SubprocessLauncher.set_environ.html

Reply:
Actually, that is what you're doing. So then there shouldn't be a problem, right?